xapi.opam

```diff
@@ -10,6 +10,7 @@ build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 
 depends: [
   "jbuilder" {build & >= "1.0+beta11"}
+  "x509"
```

Comment:
We should move it to the `x` section, as this is more or less alphabetically ordered :smile:

Reply:
Agreed. It's here at the moment only because I started out by trying to use asn1-combinators directly, then switched to x509 without moving the entry.